common/lib/xmodule/xmodule/open_ended_grading_classes/combined_open_ended_modulev1.py

```diff
@@ -567,12 +567,12 @@ def get_context(self):
             'state': self.state,
             'task_count': len(self.task_xml),
             'task_number': self.current_task_number + 1,
-            'status': ugettext(self.get_status(False)),
+            'status': ugettext(self.get_status(False)),    # pylint: disable=translation-of-non-string
             'display_name': self.display_name,
             'accept_file_upload': self.accept_file_upload,
             'location': self.location,
             'legend_list': LEGEND_LIST,
-            'human_state': ugettext(HUMAN_STATES.get(self.state, "Not started.")),
+            'human_state': ugettext(HUMAN_STATES.get(self.state, "Not started.")),    # pylint: disable=translation-of-non-string
```

Comment:
Is "Not Started" marked for translation anywhere?

If not, maybe at L562 you should add,
```python
# Mark default string so it can be scraped by our tools
default_human_state = _("Not Started.")
context = {
    ...
    'human_state': ugettext(HUMAN_STATES.get(self.state, "Not started.")),
    ...
}
```

(or something like that - whatever works best)